docs/source/gbd2017_models/causes/lower_respiratory_infections/index.rst

```diff
@@ -167,6 +160,10 @@ Data Description
      - Sources
      - Description
      - Notes
+   * - birth_prevalence_meid_1258
+     - epi
+     - Birth Prevalence of LRI
+     - get_draws('modelable_entity_id', 1258, source='epi', age_group_id=164, measure_id=5)
```

Comment:
I think we need to add birth_prevalence to the State Data table as well as the Data Sources table. I.e. state I should have measure "birth prevalence" with value `birth_prevalence_meid_1258`, and state S should have measure "birth prevalence" with measure `1 - birth_prevalence_meid_1258`.

See the neural tube defects documentation for an example: https://vivarium-research.readthedocs.io/en/latest/gbd2017_models/causes/neural_tube_defects/index.html